README.md

```diff
@@ -69,11 +69,11 @@ Check the OCI distribution conformance tests [report](https://storage.googleapis
 
 ## Compatibility
 
-The [compatibility list](https://goharbor.io/docs/2.0.0/install-config/harbor-compatibility-list/) document provides compatibility information for the Harbor components.
+The [compatibility list](https://goharbor.io/docs/2.1.0/install-config/harbor-compatibility-list/) document provides compatibility information for the Harbor components.
```

Comment:
@steven-zou We should replace the version number in these URLs with `latest` so that they will always redirect to the latest version and we dont have to update every release. `https://goharbor.io/docs/latest/install-config/harbor-compatibility-list/...`

Reply:
Good idea, I'll do change.